build.gradle.kts

```diff
@@ -0,0 +1,51 @@
+plugins {
+	java
+	id("org.springframework.boot") version "3.2.5"
+	id("io.spring.dependency-management") version "1.1.4"
+}
+
+group = "it.gov.pagopa"
+version = "0.0.1"
+
+java {
+	sourceCompatibility = JavaVersion.VERSION_17
+}
+
+configurations {
+	compileOnly {
+		extendsFrom(configurations.annotationProcessor.get())
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation("org.springframework.boot:spring-boot-starter")
+	implementation("org.springframework.boot:spring-boot-starter-web")
+	implementation("org.springframework.boot:spring-boot-starter-actuator")
+	implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.1.0")
+	compileOnly("org.projectlombok:lombok")
+	annotationProcessor("org.projectlombok:lombok")
+	testImplementation("org.springframework.boot:spring-boot-starter-test")
+
```

Comment:
add also janino dependency, used to extend the logback functionality.
Thus, add the following file used to define the template for the log produced by the application:
https://github.com/pagopa/idpay-wallet/blob/main/src/main/resources/logback-spring.xml